roles/caddy/handlers/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-- name: caddy-restart
+- name: Caddy restart
```

Comment:
Is this change triggered by the linter? I wonder, why it doesn't complain in my other repository? :thinking:

Reply:
Yeah, it's weird :( That's `name[casing]`  from https://ansible.readthedocs.io/projects/lint/rules/name/